src/main/java/edu/nitt/delta/bustracker/model/Vehicle.java

```diff
@@ -13,6 +13,8 @@
 
     private VehicleType type;
 
+    private Boolean isOccupied;
```

Comment:
It would be better to have this Boolean field in the location class rather than here, so when a driver starts sharing their location it would result in a new Location document with isOccupied `false` for erickshaws and `null` for others